example/constants/example_constants.f90

```diff
@@ -0,0 +1,25 @@
+program example_constants
+    use stdlib_constants, only: c, PI=>PI_dp
```

Comment:
Following the [style guide](https://github.com/fortran-lang/stdlib/blob/33559f551d1c7f22ea2511e4f7e779e2aed410c3/STYLE_GUIDE.md?plain=1#L39), variable names should be lowercase.
```suggestion
    use stdlib_constants, only: c, pi=>PI_dp
```
This is also consisten with the following line (`alpha =>...`)